### PR TITLE
1.20.2

### PR DESCRIPTION
**Pull request by Brokkonaut**

### CODE REVIEW

**Comment by wicivo** [comment]
> _ No description provided. _

Can you give the .jar?

**Comment by Brokkonaut** [comment]
> Can you give the .jar?

[enhancedblockentities-0.9+1.20.2.jar](https://webinterface.cubeside.de/temp/enhancedblockentities-0.9+1.20.2.jar)

File: gradle.properties [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+fabric_version=0.89.3+1.20.2
 
-mod_version = 0.9+1.20
+mod_version = 0.9+1.20.2
 maven_group = foundationgames
 archives_base_name = enhancedblockentities
 
-arrp_version=0.6.7
+arrp_version=0.8.0
 modmenu_version=7.1.0
```
Comment (MeeniMc) [inline gradle.properties]:
```suggestion
modmenu_version=8.0.0
```

**Review by MeeniMc — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (MeeniMc) [reply gradle.properties]: other than that its looking good

**Review by MeeniMc — APPROVED** [review]

**Comment by MeeniMc** [comment]
I have been using this for several weeks now and it worked flawlessly.

**Review by Earthcomputer — APPROVED** [review]
That `@ModifyVariable` is horrible and needs replacing with something else. But good enough for now.

**Comment by FoundationGames** [comment]
Looks good, thanks for contributing!
Needs a few changes but I'll take care of them after merge

**Pull request merged**